src/libraries/Microsoft.Extensions.FileProviders.Physical/src/Microsoft.Extensions.FileProviders.Physical.csproj

```diff
@@ -20,4 +26,14 @@
     <Reference Include="Microsoft.Extensions.Primitives"  />
   </ItemGroup>
 
+  <ItemGroup Condition="$(TargetFramework.StartsWith('net4'))">
+    <Reference Include="mscorlib" />
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <!-- Adding this dependency as a package reference in order to not depend on version 4.3.0.0 that
+    comes from the NetStandard.Library.NETFramework package as we want intentionally to avoid the
+    requirement on that package. -->
+    <PackageReference Include="System.Security.Cryptography.Algorithms" Version="4.3.1" Condition="'$(TargetFramework)' == 'net461'" />
```

Comment:
This comment won't be relevant once @ViktorHofer's change is in.  Let's just say 
```suggestion
    <!-- For IncrementalHash -->
    <PackageReference Include="System.Security.Cryptography.Algorithms" Version="4.3.1" Condition="'$(TargetFramework)' == 'net461'" />
```

Reply:
Sounds good, given is a comment-only change and CI is half-way through I'll let that finish and then commit this before merging.